### PR TITLE
Fix missing types

### PR DESCRIPTION
**Pull request by devcitopia**
Linked to this issue https://github.com/moleculerjs/moleculer-channels/issues/53

### CODE REVIEW

**Comment by icebob** [comment]
The problem is that the d.ts files are generated from the source code (jsdoc). So you can't modify the d.ts files directly. We should fix them in jsdoc.

https://github.com/moleculerjs/moleculer-channels/blob/2ddbb9d3ce69e769c0f81d96a2b60193215f0397/package.json#L9

**Comment by devcitopia** [comment]
You are right I just did a new commit to fix it within the JS code

File: src/index.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ const C = require("./constants");
  * @param {MiddlewareOptions} mwOpts
  * @returns Middleware
  */
-module.exports = function ChannelsMiddleware(mwOpts) {
+module.exports.default = function ChannelsMiddleware(mwOpts) {
```
Comment (icebob) [inline src/index.js:70]: It will cause breaking change.

**Review by icebob — CHANGES_REQUESTED** [review]

**Comment by icebob** [comment]
This PR contains many non-relevant changes. Please create a new one only with the relevant changes.

**Pull request closed without merging**